platform/darwin/test/MGLImageSourceTests.m

```diff
@@ -0,0 +1,42 @@
+#import <XCTest/XCTest.h>
+
+#import <Mapbox/Mapbox.h>
+
+@interface MGLImageSourceTests : XCTestCase
+
+@end
+
+@implementation MGLImageSourceTests
+
+
+- (void)testMGLImageSourceWithImageURL {
+
+    MGLCoordinateQuad quad = { { 80, 37}, { 80, 39}, { 81, 39}, { 81, 37}};
```

Comment:
That’s reasonable, although in this case the order would result in a non-quadrilateral, wouldn’t it?